CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## 0.3.0 (Feb 15, 2024)
+ - Add support for IntelliJ CE 2023.2
+ - Added custom test-project location
+ - Update the `Remote-Robot` to version 0.11.18
+ - Update Java to JDK 17
+ - Update Gradle to version 7.6
+ - Removed Redundancy in the ToolWindowsPane class
+ - Added method for reopening project
+ - Redirecting remote-robot logs to output files in home.folder/IntelliJ_debug 
```

Comment:
If #215 will be merged before the release, the location should be user.dir/intellij_debug